cola/icons.py

```diff
@@ -31,9 +32,14 @@
     '.cxx':     'file-code.svg',
 }
 
+cfg = gitcfg.current()
+style = cfg.icon_style()
 
 def install():
-    icon_dir = resources.icon_dir()
+    if not style or style == "light":
+        icon_dir = resources.icon_dir()
+    else:
+        icon_dir = resources.icon_dir(style)
```

Comment:
I think this is really the main change that I keep coming back to as a slightly unfortunate change in the relative dependencies.  Previously the `icons` module was more pure, and didn't import anything, and we didn't have to call into git so early (during module import) so that we can defer a few things.

Sorry it's taken so long to get back to this one, but it's become a little clearer with time.  Would it be okay if we made this something that's controlled via the environment instead?  

There's already precedent for a few of these in the `ENVIRONMENT VARIABLES` section of `docs/git-cola.rst`, namely `GIT_COLA_SCALE` which is directly related to the UI.  The reason these are done via the environment is that they affect the critical startup code paths that we've tried to keep as minimal as possible.

Would it be okay if we changed the style to be a lookup of `os.environ.get('GIT_COLA_THEME', 'light')`?  Presumably we can then make the `style` optional argument into a required argument, and special-case it inside of `icon_dir()` instead.

I may merge this soon with these changes applied locally, which hopefully won't be too much of a hassle to upgrade to.

Example `.bashrc`/`.xsessionrc`:

```sh
GIT_COLA_THEME=dark
export GIT_COLA_THEME
```